libmill.h

```diff
@@ -640,11 +640,15 @@ typedef struct mill_unixsock *unixsock;
 /******************************************************************************/
 
 struct mill_sslsock;
+typedef struct sslserver {
+	struct mill_ipaddr addr;
```

Comment:
Address should be a parameter to ssllisten function, independent of the context object IMO.